smarts/core/sumo_traffic_simulation.py

```diff
@@ -428,6 +401,67 @@ def sync(self, provider_state: ProviderState):
             self._reroute_vehicles(traffic_vehicle_states)
             self._teleport_exited_vehicles()
 
+    @staticmethod
```

Comment:
Why choose `staticmethod` over `property`?

Reply:
I take this back. Using`property` is also not appropriate. Another way is to store the colors as class attributes. Both approaches are fine to me.

Reply:
Exactly, class attributes would be fine as well thought. Potentially even better.